src/test/java/org/session4/homework/PersonTest.java

```diff
@@ -11,13 +11,13 @@ class PersonTest {
     @DisplayName("This person is an adult")
     void testDisplayIfThePersonIsAdult(){
         Person person = new Person(18);
```

Comment:
If I want to review just this line of code from the test, I wouldn't be able to tell what 18 stands for. I would have to check the implementation, and this takes time. We don't want to waist development time.
In this situation, we can use a constant for a person's age, with a proper name. That will point out what the value is for.
In programming, we call variables like these, `magic numbers`. We call them magic because we don't know what are used for. We should stay away from magic numbers 😄.

Reply:
![image](https://github.com/Dragos-U/itschool/assets/121547372/22b38609-58bb-4c9f-a4ee-7a7c6f97fb65)
 even if in the IDE is indicated that 18 is the age?